tests/oclinter/module-toplevel/openconfig-testcase-fail.yang

```diff
@@ -0,0 +1,35 @@
+module openconfig-testcase-fail {
+  prefix "oc-tc";
+  namespace "http://openconfig.net/linter/testcase";
+
+  import openconfig-extensions { prefix oc-ext; }
+
+  description
+    "Simple testcase to check compliance with quoting
```

Comment:
a nit, but this description seems to be the same for many of the tests, even those not checking string quoting.  something to fix later perhaps.